measures/2018/measures-data.json

```diff
@@ -31147,7 +31147,7 @@
     "metricType": "singlePerformanceRate",
     "firstPerformanceYear": 2017,
     "lastPerformanceYear": null,
-    "isHighPriority": true,
+    "isHighPriority": false,
```

Comment:
double checking cc @sarahnw

Reply:
eek how can I tell which measure this is?

Reply:
327 says false!